<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="glShowWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="xRotSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>230</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-2</number>
    </property>
    <property name="maximum">
     <number>362</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="yRotSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>60</y>
      <width>230</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-2</number>
    </property>
    <property name="maximum">
     <number>362</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="zRotSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>230</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-2</number>
    </property>
    <property name="maximum">
     <number>362</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="scaleSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>230</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="xTranSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>230</y>
      <width>230</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-10</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="yTranSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>260</y>
      <width>230</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-10</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="zTranSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>290</y>
      <width>230</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-10</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>140</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>203</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Translation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>59</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>230</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>250</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>290</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>261</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QLabel" name="xRotLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>30</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="yRotLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>60</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zRotLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>90</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scaleLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>160</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="xTranLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>230</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="yTranLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>260</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zTranLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>290</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
